libs/core/src/lib/multi-input/multi-input.component.scss

```diff
@@ -1,4 +1,6 @@
 @import "~fundamental-styles/dist/multi-input";
+@import "~fundamental-styles/dist/tokenizer";
+@import "~fundamental-styles/dist/token";
```

Comment:
Do we need it? token/tokenizer module is imported to multi-input module. These imports increases size of output file.